sdks/java/core/src/test/java/org/apache/beam/sdk/io/CompressedSourceTest.java

```diff
@@ -267,6 +371,69 @@ public void testReadMultiStreamBzip2() throws IOException {
     verifyReadContents(output, tmpFile, mode);
   }
 
+  /**
+   * Test a lzo file containing multiple streams is correctly decompressed.
+   *
+   * <p>A lzo file may contain multiple streams and should decompress as the concatenation of those
+   * streams.
```

Comment:
I think we need an enclosing </p>, or we can simply remove the <p> tag. Whichever, please be consistent with  `testReadConcatenatedLzo()`.

Reply:
This is happening during spotlessApply.